Gems/AtomLyIntegration/EMotionFXAtom/Code/Source/AtomActorDebugDraw.cpp

```diff
@@ -542,17 +542,18 @@ namespace AZ::Render
             const AZ::Vector3 bone = parentWorldPos - nodeWorldPos;
             const AZ::Vector3 boneDirection = bone.GetNormalizedEstimate();
             const AZ::Vector3 centerWorldPos = bone / 2 + nodeWorldPos;
+            const float maxBoneScale = 0.05f;
             const float boneLength = bone.GetLengthEstimate();
-            const float boneScale = CalculateBoneScale(instance, joint);
-            const float parentBoneScale = CalculateBoneScale(instance, skeleton->GetNode(parentIndex));
+            const float boneScale = AZStd::min(CalculateBoneScale(instance, joint), maxBoneScale);
+            const float parentBoneScale = AZStd::min(CalculateBoneScale(instance, skeleton->GetNode(parentIndex)), maxBoneScale);
             const float cylinderSize = boneLength - boneScale - parentBoneScale;
 
             renderColor = GetModifiedColor(color, parentIndex, cachedSelectedJointIndices, cachedHoveredJointIndex);
-            renderColor.SetA(0.75f);
+            renderColor.SetA(0.5f);
             debugDisplay->SetColor(renderColor);
```

Comment:
These want to be reversed because SetColor overwrites the Alpha channel.

Reply:
Is it? SetColor just pass the color to display, also it include the alpha channel.

Reply:
My mistake, I read it incorrectly. I was thinking you were calling `debugDisplay->SetA()`